### PR TITLE
Feature/cluster groups

### PR DESCRIPTION
**Pull request by svt-kmcdaniel**
### Description
- add endpoint support for cluster_groups <POST>
- change default accept header to application/json
- fixed exception handling within the ovc client module
- increase code coverage of ovc client module
- removed static ip address from unit test

### Issues Resolved
- resolved issue #20 

### Check List
- [x] New functionality includes testing.
  - [x] All tests pass(`$ tox`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Comment by svt-kmcdaniel** [comment]
@sijeesh 
 
I apologize for the large pull request, I was able to group the comments together based on function.

Please let me know what you think about the cluster_group implementation.

**Review by svt-mpilon — APPROVED** [review]

File: simplivity/connection.py
Comment (sijeesh) [inline simplivity/connection.py:46]: Is this tested with all the resource methods?
The reason why we made application/vnd.simplivity.v1+json as the default value is most of the simplicity REST endpoints accept this type and we don't have to pass this header all the time.

File: endpoints-support.md
Comment (sijeesh) [inline endpoints-support.md]: Is there any post operations implemented for cluster group resource?

Comment (svt-kmcdaniel) [reply endpoints-support.md]: Copy and paste error, I will change it to "GET"

File: simplivity/connection.py
Comment (svt-kmcdaniel) [reply simplivity/connection.py:46]: It appears that every new feature that is added to the rest api that wasn't in v1+ requires a custom header if we go with the 'Accept': 'application/vnd.simplivity.v1+json' such has 'cluster_groups'.  

I have talked to the rest development team and validated that all the example scripts work with this change, and have tested a get method within all the resources using the swagger UI using the 'Accept': 'application/json' and they work. 

i.e.

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/backups?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/cluster_groups?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/datastores?show_optional_fields=false&limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/external_stores?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/hosts?show_optional_fields=false&limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/omnistack_clusters?show_optional_fields=false&limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/policies?limit=500&offset=0&case=sensitive"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/certificates"

curl -X GET --header "Accept: application/json" "https://<IP_Address>/api/virtual_machines?show_optional_fields=false&limit=500&offset=0&case=sensitive"

**Review by svt-mpilon — APPROVED** [review]

**Comment by svt-kmcdaniel** [comment]
Merged into svt-kcmdaniel/simplivity-python/develop in order to unblock the internal development of our automation efforts.

Still waiting for final approval HewlettPackard/simplivity-python/master

**Review by sijeesh — APPROVED** [review]
LGTM

**Pull request merged**
